README.md

```diff
@@ -76,59 +90,59 @@ const [data, setString] = useClipboard()
 ### `useDimensions`
 
 ```js
-import { useDimensions } from 'react-native-hooks'
+import { useDimensions } from "react-native-hooks";
 
-const dimensions = useDimensions()
+const dimensions = useDimensions();
 ```
 
 ### `useGeolocation`
 
 ```js
-import { useGeolocation } from 'react-native-hooks'
+import { useGeolocation } from "react-native-hooks";
 
-const [position, stopObserving, setRNConfiguration] = useGeolocation()
+const [position, stopObserving, setRNConfiguration] = useGeolocation();
 
-console.log('latitude: ', position.coords.latitude)
+console.log("latitude: ", position.coords.latitude);
 ```
 
 ### `useNetInfo`
 
 ```js
-import { useNetInfo } from 'react-native-hooks'
+import { useNetInfo } from "react-native-hooks";
 
-const netInfo = useNetInfo()
+const netInfo = useNetInfo();
 
-console.log('netInfo type: ', netInfo.type)
+console.log("netInfo type: ", netInfo.type);
 ```
 
 ### `useKeyboard`
 
 ```js
-import { useKeyboard } from 'react-native-hooks'
+import { useKeyboard } from "react-native-hooks";
 
-const keyboard = useKeyboard()
+const keyboard = useKeyboard();
 
-console.log('keyboard show: ', keyboard.show)
-console.log('keyboard height: ', keyboard.height)
+console.log("keyboard show: ", keyboard.show);
+console.log("keyboard height: ", keyboard.height);
 ```
 
 ### `useInteractionManager`
 
 ```js
-import { useInteractionManager } from 'react-native-hooks'
+import { useInteractionManager } from "react-native-hooks";
 
-const interactionReady = useInteractionManager()
+const interactionReady = useInteractionManager();
 
-console.log('interaction ready: ', interactionReady)
+console.log("interaction ready: ", interactionReady);
```

Comment:
Can u change prettier config with single quote ?

Reply:
the script "yarn prettier --fix" does this job automatically.